### DISCUSSION TITLE
Mongodb cannot connect and create a user

### DISCUSSION THREAD

**Discussion by neerua08** [jashgopani/application-tracking-system#10] [General]
Currently, I cannot create a user without errors. Here are the logs.

![image](https://github.com/jashgopani/application-tracking-system/assets/145308018/9d757b2e-b30d-4320-94d7-bee0b701c428)

Here is the code with issues in app.py.

![image](https://github.com/jashgopani/application-tracking-system/assets/145308018/4aa08f6a-4afe-44d3-b673-593d39e20533)

**Comment by rohanajm7** [comment]
@neerua08 Please raise an issue in github issues along with the screenshots. Also, do not forget to mention how can we recreate the same.
